style.css

```diff
@@ -0,0 +1,157 @@
+.wrap {
+    max-width: 1920px;
+    min-width: 375px;
+    height: 100vh;
+    margin: 0 auto;
+    background: url(./img/backround.jpg) no-repeat;
+    background-size: cover;
+    font-family: 'Museo';
+}
+
+.header  {
+    width: 90%;
+    height: 200px;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+}
+
+.header__logo  {
+    flex: .5;
+    display: flex;
+    justify-content: left;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5898694)